### DISCUSSION TITLE
第三方app，如何通过用户获取用户的运动健康数据

### DISCUSSION THREAD

**Discussion by luluback** [zepp-health/zeppos-forum#324] [Developer Feedback]
第三方app，如何接入zepp开放平台，从而能在用户授权后获取到用户在华米下各种穿戴设备产生的健康数据，比如 步数、心率等数据

**Comment by MyoungXUE** [comment]
Hello @luluback , you can refer to this blog https://github.com/orgs/zepp-health/discussions/276

It is recommended to use English for discussion in the Github discussion area, which will help developers from all over the world.
